Documentation/flexvolume.md

```diff
@@ -5,34 +5,64 @@ indent: true
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
 
-## For Kubernetes >= 1.9.x
-In Kubernetes >= `1.9.x`, you must provide the above set Flexvolume plugin directory when deploying the [rook-operator](/cluster/examples/kubernetes/ceph/operator.yaml) by setting the environment variable `FLEXVOLUME_DIR_PATH`. For example:
+### For ContainerLinux
+Use the [Most common writeable flexvolume path](#most-common-writeable-flexvolume-path) for the next steps.
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
+
+### For OpenShift
+See the [For Atomic](#atomic) section.
+
+### For Rancher
+Rancher provides an easy way to configure kubelet. This flag can be provided to the kubelet configuration template at deployment time or by using the `up to date` feature if Kubernetes is already deployed.
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
+
+### For Tectonic
+Follow [these instructions](tectonic.md) to configure the Flexvolume plugin for Rook on Tectonic during ContainerLinux node ignition file provisioning.
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
+
+### For custom containerized kubelet
+Use the [Most common writeable flexvolume path](#most-common-writeable-flexvolume-path) for the next steps.
+
+If your kubelet is running as a (Docker, rkt, etc) container you need to make sure that this directory from the host is reachable by the kubelet inside the container.
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
+
+## Configuring the flexvolume path
+If the environment specific section doesn't mention a flexvolume path in this doc or external docs, please refer to the [Most common writeable flexvolume path](#most-common-writeable-flexvolume-path) section, before continuing to [Configuring the flexvolume path](#configuring-the-flexvolume-path).
+
+### Most common writeable flexvolume path
+The most commonly used writeable flexvolume path on most systems is `/var/lib/kubelet/volumeplugins`.
+This path is commonly used for flexvolume because `/var/lib/kubelet` is read write on most systems.
+
+### Configuring the Rook operator
+**This is only needed when using Kubernetes >= `1.9.x`**
+
+You must provide the above found flexvolume path when deploying the [rook-operator](/cluster/examples/kubernetes/ceph/operator.yaml) by setting the environment variable `FLEXVOLUME_DIR_PATH`.
```

Comment:
the operator link will be broken on the docs site since it's using a relative link that goes outside the docs tree.  it needs to be an absolute link.